### DISCUSSION TITLE
新版docker报错

### DISCUSSION THREAD

**Discussion by jayling1111** [nezhahq/nezhahq.github.io#48] [Q&A]
github.com/naiba/nezha/cmd/dashboard/controller.ServeWeb(0x400007cba0?)
	/source/cmd/dashboard/main.go:69 +0xe4
	/source/cmd/dashboard/controller/controller.go:36 +0x5b8
panic: html/template: pattern matches no files: `resource/template/**/*.html`
goroutine 1 [running]:
github.com/naiba/nezha/cmd/dashboard/controller.ServeWeb(0x40002d00c0?)
main.main()
github.com/naiba/nezha/cmd/dashboard/controller.ServeWeb(0x400058e2a0?)
github.com/naiba/nezha/cmd/dashboard/controller.ServeWeb(0x40004980c0?)

**Comment by naiba** [comment]
升级至 0.15.15
